examples/red-black-gauss-seidel.cpp

```diff
@@ -177,8 +177,8 @@ RAJA::TypedIndexSet<RAJA::ListSegment> gsColorPolicy(int N)
 
   RAJA::TypedIndexSet<RAJA::ListSegment> colorSet;
 
-  int redN = ceil(N * N / 2);
-  int blkN = floor(N * N / 2);
+  int redN = (int)ceil(N * N / 2);
```

Comment:
Better to use 'std::static_cast' for these and elsewhere since it's idiomatic C++?